drivers/virtio/net/virtio_net.c

```diff
@@ -1078,6 +1082,34 @@ static int virtio_netdev_feature_negotiate(struct uk_netdev *n,
 
 	host_features = virtio_feature_get(vndev->vdev);
 
+	/**
+	 * Large Receive Offload
+	 * NOTE: This allows the host to send packets larger than MTU. The
+	 *       network stack needs to be able to handle such packets.
+	 *       We only enable this if we have also support for mergeable RX
+	 *       buffers, otherwise we would have to either allocate huge
+	 *       buffers (wasting space for smaller buffers) or use many small
+	 *       descriptors (needing indirect descriptors and putting a large
+	 *       burden on the allocator).
+	 */
+	if (conf->lro) {
+		if (VIRTIO_FEATURE_HAS(vndev->vdev->features,
```

Comment:
Would it be worth adding a `likely` here? :thinking:

Reply:
Actually let's just reorder the whole section ^^